src/gui/CMakeLists.txt

```diff
@@ -199,6 +227,11 @@ SET(QGIS_GUI_SRCS
   layout/qgslayoutviewtooltemporarykeyzoom.cpp
   layout/qgslayoutviewtooltemporarymousepan.cpp
   layout/qgslayoutviewtoolzoom.cpp
+  layout/qgsreportfieldgroupsectionwidget.cpp
```

Comment:
Do you have a specific need for these ones to be in gui? I'd say they are still very immature and I'd rather have full flexibility to tweak them in future without the restrictions of being in the gui library.

Reply:
No I think I can do without them - I'll move them back.